fcrepo-kernel-modeshape/src/test/java/org/fcrepo/integration/kernel/modeshape/FedoraResourceImplIT.java

```diff
@@ -237,11 +242,42 @@ public void testTouch() throws RepositoryException {
         final Container obj2 = containerService.findOrCreate(session, "/" + pid);
         final FedoraResourceImpl impl = new FedoraResourceImpl(getJcrNode(obj2));
         final Instant oldMod = impl.getLastModifiedDate();
-        impl.touch();
-        assertTrue( oldMod.isBefore(obj2.getLastModifiedDate()) );
+        impl.touch(false, null, null, null, null);
+        assertTrue(oldMod.isBefore(obj2.getLastModifiedDate()));
     }
 
     @Test
+    public void testTouch() throws RepositoryException {
+        final Calendar forgedDate = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
```

Comment:
That's fair enough.  I'll replace this with language that doesn't imply malfeasance.